src/segments/cds.go

```diff
@@ -0,0 +1,68 @@
+package segments
+
+import (
+	"encoding/json"
+	"oh-my-posh/environment"
+	"oh-my-posh/properties"
+)
+
+type Cds struct {
+	language
+	HasDependency bool
```

Comment:
This isn't present in the docs template properties it seems

Reply:
Could you point to similar case in existing segments? Not sure if I understand correctly.

Reply:
Added a comment above.